src/libsystemd-network/sd-dhcp-server-lease.c

```diff
@@ -215,7 +215,8 @@ static int dhcp_server_lease_append_json(sd_dhcp_server_lease *lease, JsonVarian
                                  JSON_BUILD_PAIR_BYTE_ARRAY("ClientId", lease->client_id.raw, lease->client_id.size),
                                  JSON_BUILD_PAIR_IN4_ADDR_NON_NULL("Address", &(struct in_addr) { .s_addr = lease->address }),
                                  JSON_BUILD_PAIR_STRING_NON_EMPTY("Hostname", lease->hostname),
-                                 JSON_BUILD_PAIR_FINITE_USEC("ExpirationUSec", lease->expiration)));
+                                 JSON_BUILD_PAIR_FINITE_USEC("ExpirationUSec", lease->expiration.boottime),
+                                 JSON_BUILD_PAIR_FINITE_USEC("ExpirationTimestamp", lease->expiration.realtime)));
```

Comment:
I#d call this ExpirationRealtimeUSec, i.e. emphasize what this is, and linux/posix calls this CLOCK_REALTIME after all.